### PR TITLE
add TuneTag 1.2

### PR DESCRIPTION
**Pull request by piijai**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/t/tunetag.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask "tunetag" do
+  version "1.2"
+  sha256 :no_check
+
+  url "https://sweetpproductions.com/products/tunetag/TuneTag.dmg"
+  name "TuneTag"
+  desc "Is a lightning fast ID3 and meta data editor for mp3 and m4a audio files"
```
Comment (bevanjkay) [inline Casks/t/tunetag.rb:7]:
```suggestion
  desc "ID3 and metadata editor for audio files"
```

File: Casks/t/tunetag.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  auto_updates true
+  depends_on macos: ">= :big_sur"
+
+  app "TuneTag.app"
+
+  zap trash: [
+    "~/Library/Containers/com.sweetpproductions.TuneTag",
+    "~/Library/Group Containers/U928YCMMNK.com.sweetpproductions.TuneTag",
```
Comment (bevanjkay) [inline Casks/t/tunetag.rb:22]:
```suggestion
    "~/Library/Group Containers/*.com.sweetpproductions.TuneTag",
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Comment by piijai** [comment]
@bevanjkay Thanks for your requested changes.

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @piijai

**Pull request merged**